react/components/Modal/modal.css

```diff
@@ -26,3 +18,60 @@
 .scrollBar::-webkit-scrollbar-button {
   display: none;
 }
+
+.contents {
+  display: contents;
+}
+
+.hiddenScroll {
+  overflow: hidden;
+  padding-right: 15px;
+}
+
+.mh60 {
+  max-height: 80vh;
+}
+
+@media screen and (min-width: 40em) {
+  .mw30 {
+    max-width: 30rem;
+  }
+}
+
+@media screen and (min-width: 40em) {
+  .mw40 {
+    max-width: 40rem;
+  }
+}
+
+@media screen and (min-width: 40em) {
+  .mw50 {
+    max-width: 50rem;
+  }
+}
```

Comment:
I don't think we should create classes like tachyons, because if we decide to add some handle like this in the tachyons this one will have more priority

Reply:
And I think you should try to use tachyons instead of writing these values

Reply:
Tachyons don't have `max-width`, but I will change the names as you suggested

Reply:
It does https://github.com/vtex/vtex-tachyons/blob/master/tachyons.css#L1370

Reply:
> Tachyons don't have `max-width`, but I will change the names as you suggested

It does https://github.com/vtex/vtex-tachyons/blob/master/tachyons.css#L1370 @thayannevls

Reply:
Done! 

http://5d6fca40.ngrok.io/?path=/story/components-modal--default

Reply:
Sorry @Klynger , I was trying to say that tachyons don't have the size that the designer requested for the Modal sizes :/

Reply:
Yeah, but in this case shouldn't the designer reconsider maybe? Because those sizes seem like customizations of the app that the user would do

Reply:
I'll discuss with @oldufre about this but if we take longer to decide about this, do you think it's okay to merge with these classes and maybe change later?

Reply:
The store isn't using this component (because if it was, this would be a breaking change), so I think it's ok

Reply:
Hey @Klynger , I just spoke with @oldufre and she confirmed about the current sizes are the ideal for the Modal. We also showed these sizes in the Styleguide Meeting and everyone agreed too